google-cloud-logging/src/main/java/com/google/cloud/logging/MonitoredResourceUtil.java

```diff
@@ -212,6 +223,11 @@ private static List<LoggingEnhancer> createEnhancers(Resource resourceType) {
         enhancers.add(
             new LabelLoggingEnhancer(APPENGINE_LABEL_PREFIX, ImmutableList.of(Label.InstanceName)));
       }
+    } else if (resourceType == Resource.CLOUD_RUN_JOB) {
+      enhancers.add(
+          new LabelLoggingEnhancer(
+              CLOUD_RUN_JOB_LABEL_PREFIX,
+              ImmutableList.of(Label.CloudRunJobExecutionName, Label.CloudRunJobTaskIndex)));
```

Comment:
According to this (https://cloud.google.com/logging/docs/api/platform-logs#cloud_run) you're missing a label for task attempt.

Reply:
Good catch. Added cloud run task attempt label into enhancer.